lib/core_extensions.rb

```diff
@@ -140,17 +140,20 @@ def to_translation
   end
 
   def to_gb
-    value, _, unit=self.match(/(\d+(\.\d+)?) ?(([KMGT]i?B?|B))$/i)[1..3]
+    match_data = self.match(/(\d+(\.\d+)?) ?(([KMGT]i?B?|B|Bytes))$/i)
+    if match_data.present?
+      value, _, unit = match_data[1..3]
+    else
+      raise "Unknown string: #{self.inspect}!"
+    end
     case unit.to_sym
```

Comment:
we use case insensitive matching, so for example "10gb" will be caught, but there is no `:gb` symbol - perhaps `.lower.to_sym`?